packages/components/src/components/button-link/button-link.tsx

```diff
@@ -1,9 +1,16 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-const ButtonLink = ({ children, className, to, onClick, size = 'medium' }) => (
+type TButtonLinkProps = {
+    children: React.ReactNode;
+    className: string;
+    onClick: () => void;
+    size: string;
+    to: string;
+};
```

Comment:
@jim-deriv great job, just one nit suggestion to shorten the type 😊:
```suggestion
type TButtonLinkProps = React.ComponentPropsWithoutRef<"a"> & {
    size: string;
    to: string;
};
```